Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -50,6 +50,16 @@ class SearchFilter extends AbstractFilter
     const STRATEGY_WORD_START = 'word_start';
 
     /**
+     * @var string Doctrine expression to case insensitivity comparison
+     */
+    const CASE_INSENSITIVE = 'LOWER';
+
+    /**
+     * @var bool whether it's case sensitive or not
+     */
+    private $caseSensitive = true;
```

Comment:
I indeed assumed that everytime the filter is used it'll go through `apply` before using the `addWhereByStrategy`. At first I indeed use the "case sensitive" information as argument but then it just didn't feel right.
I could avoid the property but then it's lot of repetitions for not so much, you tell me ;).